src/translations/index.js

```diff
@@ -105,6 +105,12 @@ const TRANSLATIONS = {
         'remove-key-confirm': 'Log out of your wallet',
         'remove-key-recovery-words': 'Recovery Words',
         'remove-key-back': 'Back to logout',
+
+        'derive-address-heading-passphrase': 'Unlock your Wallet',
```

Comment:
```suggestion
        'derive-address-heading-passphrase': 'Unlock your wallet',
```
Because `wallet` is not a name.